src/Config/ObjectModel/Entity.cs

```diff
@@ -34,6 +34,8 @@ public record Entity
     public Dictionary<string, EntityRelationship>? Relationships { get; init; }
     public EntityCacheOptions? Cache { get; init; }
 
+    public EntityHealthCheckConfig? Health { get; init; } = new();
```

Comment:
If you notice, when `cache` is not defined. we do allow null to be specified. Its just handled in code to pick up the default value. If we always instantiate Health, even if its not specified in the config - then we will end up writing it back to the file system. This will lead to a side effect of minimal configs getting bloated even if not intended. This has been a cause of bug when we implemented optional properties previously. Please refer to those properties as examples on how this was done. Let me know if you have questions on this.